yarn.lock

```diff
@@ -2596,6 +2608,15 @@
     "@graphql-tools/utils" "^8.8.0"
     tslib "~2.4.0"
 
+"@graphql-codegen/schema-ast@^3.0.1":
```

Comment:
we're importing 2 copies of `@graphql-codegen/schema-ast` can this be condensed?